# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/boost/hotfix-siglost.patch.alpha
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- boost_1_86_0/libs/process/src/error.cpp.vanilla	2024-10-16 13:14:08.816673784 +0200
+++ boost_1_86_0/libs/process/src/error.cpp	2024-10-16 13:14:27.356674241 +0200
@@ -100,7 +100,7 @@
 #            if defined(SIGILL)
                     case SIGILL: return "SIGILL:    Illegal Instruction";
 #            endif
-#            if defined(SIGINFO)
+#            if 0 && defined(SIGINFO)
                     case SIGINFO: return "SIGINFO:   A synonym for SIGPWR";
 #            endif
 #            if defined(SIGINT)
